tools/ci/docker/engine/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/graalvm/jdk-community:21
+FROM ghcr.io/graalvm/jdk-community:24
```

Comment:
Isn't the current JDK 24.0.1?

Reply:
Note that `24` may "expand" to `24.0.0`. And yes, the current JDK is **24.0.1**.

Reply:
Yes, it will expand to latest, most-specific one. We didn't have this for 21.* so I didn't see why change it now.